blocks/article-header/article-header.css

```diff
@@ -19,26 +19,26 @@
     ) {
     max-width: var(--page-constrained-text);
     margin-inline: auto;
-  }
-
-  h1,
-  h2,
-  h3,
-  h4,
-  h5,
-  h6 {
-    margin-block-start: var(--spacing-heading-block-start-small-screen);
-
-    @media (min-width: 22.5rem) {
-      margin-block-start: var(--spacing-heading-block-start-medium-screen);
-    }
-
-    @media (min-width: 48rem) {
-      margin-block-start: var(--spacing-heading-block-start-large-screen);
-    }
 
-    @media (min-width: 80rem) {
-      margin-block-start: var(--spacing-heading-block-start-xl-screen);
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      margin-block-start: var(--spacing-heading-block-start-small-screen);
+
+      @media (min-width: 22.5rem) {
+        margin-block-start: var(--spacing-heading-block-start-medium-screen);
+      }
+
+      @media (min-width: 48rem) {
+        margin-block-start: var(--spacing-heading-block-start-large-screen);
+      }
+
+      @media (min-width: 80rem) {
+        margin-block-start: var(--spacing-heading-block-start-xl-screen);
+      }
```

Comment:
spacing only happens within the `article` tag to prevent spacing in the article footer